### ISSUE TITLE
Bias-factorized ChromBPNet training still running after 6 days

### ISSUE THREAD

**Issue by Manonbaudic** [kundajelab/chrombpnet#228]
Hello,
I am running ChromBPNet on ATAC-seq data from ENCODE (ENCFF884PIS). I successfully generated a custom bias model (bias.h5), which took slightly over 24 hours to complete. However, when running the Bias-Factorized ChromBPNet with the pre-trained bias model, I’ve noticed the process is taking an exceptionally long time.
After 6 days, it is still running, and the log file indicates that I am currently at step 11 out of 50. Do you know why this might be taking so long or if there are any optimizations I can apply to speed up the process?
"Epoch 11/50
 364/1595 [=====>........................] - ETA: 10:43:08 - loss: 348.1603 - logits_profile_predictions_loss: 341.5599 - logcount_predictions_loss: 0.4963." 

Here is the job I submitted: 
"#!/bin/bash -l
cd /path/chrombpnet/
source /path/mambaforge/bin/activate chrombpnet
chrombpnet pipeline -ibam /path/ENCFF884PIS.bam -d "ATAC" -g /path/chrombpnet/genome.fa -c /path/genome_files/chrom.sizes/mm10.chrom.sizes -p /path/chrombpnet/ENCFF767MGH.bed -n /path/chrombpnet/output_nonpeaks/output_negatives.bed -fl /path/chrombpnet/folds/fold_0.json -b /path/chrombpnet/output_bias-model-training/models/bias.h5 -o /path/chrombpnet/output_ChromBPNet_training/"

Thank you very much for our help, 

Manon

**Comment by akundaje** [comment]
It is very likely you aren't using a GPU. Something may be off with your
configuration. It should definitely not take 24 hours to train a bias model.

**Comment by panushri25** [comment]
@Manonbaudic Yes I dont think the job is using the GPU.

**Comment by panushri25** [comment]
Closing due to inactivity. Please feel free to open again.
